### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by JustinDrake**

### CODE REVIEW

File: specs/beacon-chain.md [hunk 1]
```diff
@@ -502,8 +502,8 @@ For all (`shard_id`, `shard_block_hash`) tuples, compute the total deposit size
 Let `time_since_finality = block.slot - last_finalized_slot`, and let `B` be the balance of any given validator whose balance we are adjusting, not including any balance changes from this round of state recalculation. Let:
 
 * `total_deposits = sum([v.balance for i, v in enumerate(validators) if i in get_active_validator_indices(validators, current_dynasty)])` and `total_deposits_in_ETH = total_deposits // 10**18`
-* `reward_quotient = BASE_REWARD_QUOTIENT * int_sqrt(total_deposits_in_ETH)` (1/this is the per-slot max interest rate)
-* `quadratic_penalty_quotient = SQRT_E_DROP_TIME**2` (after D slots, ~D<sup>2</sup>/2 divided by this is the portion lost by offline validators)
+* `reward_quotient = BASE_REWARD_QUOTIENT * int_sqrt(total_deposits_in_ETH)` (this is the per-slot max interest rate)
```
Comment (djrtwo) [inline specs/beacon-chain.md]: "one over this number is the per-slot max interest rate" -- as in `1 / reward_quotient`

Comment (djrtwo) [reply specs/beacon-chain.md]: the original language is unclear in its current form but correct

Comment (JustinDrake) [reply specs/beacon-chain.md]: I see. Will edit

**Comment by djrtwo** [comment]
Good to go 👍

**Pull request merged**